tests/plugins/balloonpanel/positioning.js

```diff
@@ -80,19 +80,18 @@
 				if ( !this._getFrameMethodReplaced ) {
 					// The problem is also window.getFrame().getClientRect() as it returns different results from dashboard and directly.
 					this._getFrameMethodReplaced = true;
-					var orig = this.editor.window.getFrame;
-
-					this.editor.window.getFrame = function() {
-						var ret = orig.call( this );
-
-						if ( ret ) {
-							ret.getClientRect = function() {
-								return { height: 300, width: 300, left: 1, bottom: 643, right: 301, top: 343 };
-							};
-						}
-
-						return ret;
-					};
+					var editor = this.editor,
+						rect = { height: 300, width: 300, left: 1, bottom: 643, right: 301, top: 343 },
+						frame = editor.window.getFrame(),
+						container = frame.getParent();
+
+					sinon.stub( editor.window, 'getFrame' ).returns( frame );
+					sinon.stub( CKEDITOR.env.safari ? container : frame, 'getClientRect' ).returns( rect );
+
+					if ( CKEDITOR.env.safari ) {
+						// Override container because iframe has wrong rect values in mobile Safari (#1076).
```

Comment:
Again, comment mentions mobile Safari while the `if` condition is for both desktop and mobile ones.

Reply:
Same reason as https://github.com/ckeditor/ckeditor-dev/pull/2819#discussion_r268246828